runelite-client/src/main/java/net/runelite/client/plugins/bank/BankPlugin.java

```
@@ -0,0 +1,234 @@
+
+		if (widgetItemContainer == null) return widgetItems;
+
+		Widget[] widgetChildren = widgetItemContainer.getDynamicChildren();
+
+		for (int i = 0; i < widgetChildren.length; i++)
+		{
+			if (widgetChildren[i].getItemId() == ITEM_EMPTY || widgetChildren[i].getItemId() <= 0 || widgetChildren[i].isHidden())
+				continue;
+
+			widgetItems.add(widgetChildren[i]);
+		}
+
+		return widgetItems;
+	}
+
+	private String formatNumber(double n)
+	{
+		return formatNumber(n, 0);
+	}
+
+	private String formatNumber(double n, int iteration)
+	{
+		if (n >= 1000)
+		{
+			return formatNumber(((int) (n / 100D)) / 10D, iteration + 1);
+		}
+		else
+		{
+			return ((int) n) + suffixes[iteration];
+		}
+	}
+
+	public class BankRunnable implements Runnable
+	{
+		private boolean boolExit = false;
+
+		public void run()
+		{
+			try
+			{
+				boolExit = false;
+
+				Widget widgetBankTitleBar = client.getWidget(WidgetInfo.BANK_TITLE_BAR);
+
+				if (widgetBankTitleBar == null || widgetBankTitleBar.isHidden()) return;
+
+				java.util.List<Widget> widgetItems = GetItems();
+
+				if (widgetItems.size() <= 0) return;
+
+				client.getWidget(WidgetInfo.BANK_ITEM_COUNT).setText(widgetItems.size() + "");
+
+				int intGEValue = 0;
+				int intHAValue = 0;
+
+				for (Widget widget : widgetItems)
+				{
+					if (boolExit) return;
+
+					if (widget.getItemQuantity() <= 0 || widget.getItemId() == -1) continue;
+
+					int intPrice[] = getItemPrices(widget.getItemId(), widget.getItemQuantity());
+
+					if (config.includeUntradeable() && intPrice[0] == 0) intPrice[0] = intPrice[1];
+
+					intGEValue += intPrice[0];
+					intHAValue += intPrice[1];
+
+					strCurrentTab = config.customName() + " ~" + formatNumber(intGEValue) + " (GE)";
+
+					if (config.showHA()) strCurrentTab += " ~" + formatNumber(intHAValue) + " (HA)";
+
+					widgetBankTitleBar.setText(strCurrentTab);
+				}
+			} catch (Exception e)
+			{
+
```

Never use empty catch statements